src/styles.ts

```diff
@@ -336,17 +362,32 @@ const applyDimensionStyles = (node: YogaNode, style: Styles): void => {
 	}
 };
 
-const applyDisplayStyles = (node: YogaNode, style: Styles): void => {
+const applyDisplayStyles = (
+	domNode: DOMNode & {yogaNode: YogaNode},
+	style: Styles
+): void => {
+	const node = domNode.yogaNode;
+
 	if ('display' in style) {
 		node.setDisplay(
 			style.display === 'flex' ? Yoga.DISPLAY_FLEX : Yoga.DISPLAY_NONE
 		);
 	}
 };
 
-const applyBorderStyles = (node: YogaNode, style: Styles): void => {
+const applyBorderStyles = (
+	domNode: DOMNode & {yogaNode: YogaNode},
+	style: Styles
+): void => {
+	const node = domNode.yogaNode;
+
 	if ('borderStyle' in style) {
-		const borderWidth = typeof style.borderStyle === 'string' ? 1 : 0;
+		const borderWidth =
+			domNode.nodeName === 'ink-line'
```

Comment:
I needed access to `nodeName` here, so I figured I'd update all the applyXYZ method to take the DOM node instead.